activation/handler.go

```diff
@@ -483,32 +483,45 @@ func (h *Handler) checkWrongPrevAtx(
 		return nil, fmt.Errorf("%s referenced incorrect previous ATX", atx.SmesherID.ShortString())
 	}
 
-	// check if atx.PrevATXID is actually the last published ATX by the same node
-	prev, err := atxs.Get(tx, prevID)
-	if err != nil {
-		return nil, fmt.Errorf("get prev atx: %w", err)
-	}
-
-	// if atx references a previous ATX that is not the last ATX by the same node, there must be at least one
-	// atx published between prevATX and the current epoch
 	var atx2 *types.VerifiedActivationTx
-	pubEpoch := h.clock.CurrentLayer().GetEpoch()
-	for pubEpoch > prev.PublishEpoch {
-		id, err := atxs.PrevIDByNodeID(tx, atx.SmesherID, pubEpoch)
+	if atx.PrevATXID == types.EmptyATXID {
+		// if the ATX references an empty previous ATX, we can just take the initial ATX and create a proof
+		// that the node referenced the wrong previous ATX
+		id, err := atxs.GetFirstIDByNodeID(tx, atx.SmesherID)
 		if err != nil {
-			return nil, fmt.Errorf("get prev atx id by node id: %w", err)
+			return nil, fmt.Errorf("get initial atx: %w", err)
```

Comment:
NIT, to distinguish this error from the following:
```suggestion
			return nil, fmt.Errorf("get initial atx id: %w", err)
```